src/Magnum/Audio/CMakeLists.txt

```diff
@@ -45,6 +45,11 @@ set(MagnumAudio_HEADERS
 
     visibility.h)
 
+if(WITH_SCENEGRAPH)
```

Comment:
Yeah, I figured you would obviously want to have Audio without SceneGraph sometimes.